pom.xml

```diff
@@ -493,6 +494,7 @@
         <version.org.eclipse.microprofile.reactive-streams-operators.api>3.0</version.org.eclipse.microprofile.reactive-streams-operators.api>
         <legacy.version.org.eclipse.microprofile.rest.client.api>2.0</legacy.version.org.eclipse.microprofile.rest.client.api>
         <version.org.eclipse.microprofile.rest.client.api>3.0</version.org.eclipse.microprofile.rest.client.api>
+        <version.org.eclipse.microprofile.telemetry>1.0-RC7</version.org.eclipse.microprofile.telemetry>
```

Comment:
The ```1.0``` is available - does it make sense to upgrade for the release?

Reply:
Updated.